proposals/0000-haskell-symposium-changes.md

```diff
@@ -0,0 +1,51 @@
+
+One contributing explanation for this decline is the increasing role bibliometrics plays in evaluating career academics.  Despite the quality of reviewing at the Haskell Symposium, it is listed as a C tier venue—"other ranked conference venues that meet basic standards for peer reviewed venues"—in the CoRE rankings, and is not included in the [csrankings.org](https://csrankings.org) rankings of computer science departments.
+
+In response, the Steering Committee is proposing two changes to the Haskell Symposium.  They are each, independently, intended to restore the Symposium's role as a center of vibrant discussion in research on pure functional programming.
+
+## Proposal 1: Talk Proposals
+
+The steering committee proposes that the Haskell Symposium invite a new category of submissions, **talk proposals**. Including talk proposals invites discussion of high-quality work at the Haskell Symposium, while recognizing the pressures that lead that work to be preferentially published at a future SIGPLAN conference.
+
+Talk proposals will be described in the call for papers by:
+
+> Talk proposals need not be full-length, and should report work in progress relevant to Haskell language design, theory, tools, or applications.  Extended abstracts will be evaluated by the PC for novelty and relevance to the Haskell community, but are not expected to include finished results.  
+
+Talk proposals will not be distributed to attendees, but authors of talk proposals may provide links to materials to be included on the program.
+
+## Proposal 2: JFP Submission of Accepted Papers
+
+The steering committee proposes to cease publishing the proceedings of the Haskell Symposium as an archival ACM publication.   Instead, **authors of accepted research papers and functional pearls will have the option of submitting their papers to an expedited review process for inclusion in the Journal of Functional Programming**. This provides a better-regarded publication venue for work presented at the Haskell Symposium, encouraging submission of high quality work to the Symposium.
+
+### For authors
+
+Under this proposal, authors will still submit papers to the Haskell Symposium, and receive reviews from the PC, on the usual schedule.    
+
+Authors of accepted papers will present their papers at the Symposium.  
+
+JFP will accept the HS reviews as the first round of reviewing for JFP, provided the reviews meet the standards of the journal.  Authors of accepted papers will then be given the option to revise their papers for submission to the Journal of Functional Programming.  To assure that these appear in a timely fashion, the deadline for revision will likely be around the time of the Symposium itself.  Revised paper will be reviewed by JFP, most likely by the original reviewers.
```

Comment:
I find that attractive.

Should the deadline be “soon after the Symposium”, so that any insights gained from both the last-minute talk preparation and especially the hallway discussions after the talk or other timely feedback from attendees can be incorporated?

Reply:
I also would like this. I'd suggest that the deadline be about 3-4 weeks after the Symposium.

Why it should be after:
- We'll gather more feedback during the conference, which can help make the paper better.
- People sometimes are busy during the weeks prior preparing their slides and getting funding and making travel arrangements, so having it at the same time can be a pain.

Why 3-4 weeks and not just 3-4 days:
- Less risk of people having to spend their ICFP week working on a paper for submission to JFP.
- We won't don't have to cut the trip short or rush on the way back. It's not unusual to take time off right after and to have some catching up to do at work after coming back from ICFP.

Reply:
> The steering committee proposes to cease publishing the proceedings of the Haskell Symposium as an archival ACM publication. [...]

> For authors
> 
> Under this proposal, authors will still submit papers to the Haskell Symposium, and receive reviews from the PC, on the usual schedule.
> 
> Authors of accepted papers will present their papers at the Symposium.

Can we really call them accepted "papers" anymore if there are no proceedings? Aren't they more like "accepted talks"?

> JFP will accept the HS reviews as the first round of reviewing for JFP, provided the reviews meet the standards of the journal. Authors of accepted papers will then be given the option to revise their papers for submission to the Journal of Functional Programming.
> 
> The final decision of the JFP editors is, of course, independent of the decision of the Haskell Symposium PC.

Could this make universities, employers and funding organizations not fund trips for submitters because there's no paper being published at the Symposium?

I'd like to propose an alternative: the same that ICFP does. Publish the symposium papers perhaps in a PACML-like journal, and invite selected papers to publish extended versions at JFP.

Reply:
Also, the benefit of doing it this way is that, by the time of the conference, there's a published paper that people can read.

What you are proposing is effectively to have post-proceedings, which is what IFL and TFP do. I don't submit there because they have post-proceedings, and there's zero guarantee of the paper being published, so as an author/employer, we acquire the cost up-front for a possible return.

Many funding organizations take paper publications as a measure of productivity. If somebody submits to the Haskell Symposium, presents the paper, and then JFP says "no" to that paper, the people who funded their trip may be livid, and they authors may have spent their whole travel budget on this event.

Another alternative would be to have the submission deadline _way earlier_, so that, by the time that the author gets to the symposium, the paper is accepted _and_ published.

Reply:
I agree with Ivan that authors may not be keen on presenting a paper that isn't guaranteed to lead to a publication.  There's a danger that not getting a guaranteed publication makes the submission rate worse, rather than better.  One possible solution is to use a two-phase reviewing process prior to the paper being accepted for the Haskell Symposium and JFP publication, which is how special issues of the journal tend to be run, e.g. http://tinyurl.com/mr37duxh.  But the downside is that this process takes more time, although it could be compressed for the purposes of the symposium rather than a special issue.